diff --git a/dist/config.mjs b/dist/config.mjs
index 93df946165827ce510c436020e46ecf7ffb1219a..fb25622902c40889040e1d8767b2ccd7cebc51f2 100644
--- a/dist/config.mjs
+++ b/dist/config.mjs
@@ -190,7 +190,9 @@ async function getVitestConfigFromNuxt(options, loadNuxtOptions = {}) {
 	return resolvedConfig;
 }
 async function defineVitestProject(config) {
-	return await resolveConfig(defu$1({ test: { environment: "nuxt" } }, config));
+	const resolvedConfig = await resolveConfig(defu$1({ test: { environment: "nuxt" } }, config));
+	resolvedConfig.extends ??= false;
+	return resolvedConfig;
 }
 const defineViteConfig = (config) => config;
 function defineVitestConfig(config = {}) {
diff --git a/dist/vitest-environment.mjs b/dist/vitest-environment.mjs
index 01016dcc82d2735858ba828720176e8a265474a7..1a3b996083aabf77aac697ff51bc3640ac8de6da 100644
--- a/dist/vitest-environment.mjs
+++ b/dist/vitest-environment.mjs
@@ -234,14 +234,14 @@ var vitest_default = {
 		return { teardown() {
 			keys.forEach((key) => delete global[key]);
 			teardownWindow();
-			originals.forEach((v, k) => global[k] = v);
+			originals.forEach((descriptor, k) => Object.defineProperty(global, k, descriptor));
 			if (!global.IntersectionObserver) global.IntersectionObserver = IntersectionObserver;
 			teardown();
 		} };
 	}
 };
 async function importVitestEnvironments() {
-	return await (resolveModulePath("vitest/runtime", { try: true }) ? import("vitest/runtime") : import("vitest/environments"));
+	return await import("vitest/runtime");
 }
 var IntersectionObserver = class {
 	observe() {}
diff --git a/package.json b/package.json
index 69e1c08716e7f98e42604e058c95aa9e7496c133..909d4484f917b1c2ad15d948ce730719a90f2aa1 100644
--- a/package.json
+++ b/package.json
@@ -136,7 +136,7 @@
     "happy-dom": ">=20.0.11",
     "jsdom": ">=27.4.0",
     "playwright-core": "^1.43.1",
-    "vitest": "^4.0.2"
+    "vitest": "^5.0.0"
   },
   "peerDependenciesMeta": {
     "@cucumber/cucumber": {
